lib/google_contacts_api/contacts_accessor.rb

```diff
@@ -25,5 +25,37 @@ def get_contacts(params = {})
       end
       GoogleContactsApi::ContactSet.new(response.body, @api)
     end
+
+    def post_contact(contact)
+      params = {}
+      headers = {
+          'Content-type' => 'application/json'
+          }
+      body = contact.json
+      params[:body] = body
+      url = "contacts/default/full"
+      response = @api.post_v2(url, params, headers)
+      GoogleContactsApi::Api.parse_response_code(response)
+    end
+
+    def put_contact(contact)
+      params = {}
+      headers = {
+          'Content-type' => 'application/json',
+          'If-Match' => '*'
```

Comment:
The If-Match = "*" header currently doesn't seem to be noticed when the gem is used by a call that requires it.  I'm still working on that, but I'm also trying to work out if it better to allow any headers or parameters in when a call is made to the gem to save a contact/group.